build.proj

```diff
@@ -166,6 +167,14 @@
 
   <!-- Solution Targets -->
 
+  <Target
+    Name="CheckRestore"
+    BeforeTargets="Build">
+    <Error
+      Condition="!Exists('$(RestoreStampFile)')"
+      Text="When building with MSBuild directly, The 'Restore' target must be run first. Run 'msbuild /restore' or 'msbuild /t:Restore'."/>
```

Comment:
Maybe clarify here that `msbuild /restore` still runs the `Build` target?

Reply:
Nah. It's pretty obvious.